using-timescaledb/telemetry.md

```diff
@@ -79,15 +79,26 @@ version warnings. While you do not have to update immediately to the newest
 release, many users have reported that performance issues or bugs
 automatically resolve after updating their version of TimescaleDB.
 
-## Disabling Telemetry
+## Disabling Telemetry [](telemetry-disable)
+
 Although we invite our community to help us keep improving our
-product, we do understand when users would like to disable telemetry. Note that
-disabling telemetry also disables the version checking functionality.
+product, we do understand when users would like to disable
+telemetry. Note that disabling telemetry also disables the version
+checking functionality.
+
+Telemetry is sent on a per-database basis, so users can disable
+telemetry for specific databases or for an entire instance.
 
-Telemetry is sent on a per-database basis, so users can disable telemetry for specific databases or for an entire instance.
+To disable telemetry, it is necessary to both remove the telemetry job
```

Comment:
This is a bug with our system. I strongly think we should not encourage users to modify catalog tables. In 2.0 we will have the ability to disable jobs.  Meanwhile, we have a couple of other options : a) A temporary bug fix might be to actually delete the job from the table if a user disables telemetry or b) Document the work around as part of the github issue but not as part of the documentation.